plugin-fancymenu/lxqtfancymenuwindow.cpp

```diff
@@ -207,6 +209,7 @@ LXQtFancyMenuWindow::LXQtFancyMenuWindow(QWidget *parent)
     mCategoryModel->setAppMap(mAppMap);
     mCategoryView->setModel(mCategoryModel);
 
+    connect(mAppModel, &LXQtFancyMenuAppModel::favoritesChanged, this, &LXQtFancyMenuWindow::favoritesChanged);
```

Comment:
This doesn't have any effect because `mFavorites` remains the same. You could do as follows:

First, you could change the signal `LXQtFancyMenuAppModel::favoritesChanged()` to
```c++
    void favoritesReordered(int oldPos, int newPos);
```
and use it in `LXQtFancyMenuAppModel::dropMimeData`. Then, change the above connection to

```c++
    connect(mAppModel, &LXQtFancyMenuAppModel::favoritesReordered, this, [this] (int oldPos, int newPos) {
        mFavorites.move(oldPos, newPos);
        emit favoritesChanged();
    });
```

Reply:
Done by entirely removing `mFavorites`